### PR TITLE
qt: add vulkan support

### PR DESCRIPTION
**Pull request by makigumo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/qt.rb [hunk 1]
```diff
@@ -22,6 +22,7 @@ class Qt < Formula
   depends_on "pkg-config" => :build
   depends_on :xcode => :build
   depends_on :macos => :sierra
+  depends_on "molten-vk"
```
Comment (fxcoudert) [inline Formula/qt.rb]: Is this a build-time or runtime dependency?

File: Formula/qt.rb [hunk 2]
```diff
@@ -40,6 +41,8 @@ def install
       -pkg-config
       -dbus-runtime
       -proprietary-codecs
+      -I#{Formula["molten-vk"].opt_include}
```
Comment (fxcoudert) [inline Formula/qt.rb]: All other packages are autodetected, why are the molten-vk headers not autodetected?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/qt.rb (hunk 1 above)
Comment (makigumo) [reply Formula/qt.rb]: A Qt application that is not using Vulkan will run without molten-vk installed. If that is what you are asking.

File: Formula/qt.rb (hunk 2 above)
Comment (makigumo) [reply Formula/qt.rb]: I'm not sure. I followed this blog post https://www.qt.io/blog/2018/05/30/vulkan-for-qt-on-macos 
Maybe only the path to molten-vk’s vulkan headers in `libexec/include` is required. I’m esting this locally now.

**Pull request closed without merging**